tests/library/page-clock.spec.ts

```diff
@@ -437,8 +437,7 @@ it.describe('while running', () => {
     await page.clock.install({ time: 0 });
     await page.goto('data:text/html,');
     await page.clock.pauseAt(1000);
-    await page.waitForTimeout(1000);
-    await page.clock.resume();
+    await page.waitForTimeout(2000); // Should not progress time.
```

Comment:
why did this change?

Reply:
The test looked wrong to me, it was testing that the paused time at 1000 corresponds to the `Date.now()` call. For some unknown reason it resumed the clock afterwards. Was this intentional?

Reply:
Extracted into https://github.com/microsoft/playwright/pull/35522.